.github/workflows/tests.yaml

```diff
@@ -32,12 +32,12 @@ jobs:
         id: cache
         with:
           path: ${{ env.pythonLocation }}
-          key: ${{ runner.os }}-${{ runner.arch }}-Python-${{ steps.setup-python.outputs.python-version }}-${{ hashFiles('requirements/required.txt', 'requirements/datasets.txt', 'requirements/tests.txt') }}
+          key: ${{ runner.os }}-${{ runner.arch }}-Python-${{ steps.setup-python.outputs.python-version }}-${{ hashFiles('requirements/required.txt', 'requirements/datasets.txt', 'requirements/models.txt', 'requirements/tests.txt') }}
         if: ${{ runner.os != 'macOS' }}
       - name: Install pip dependencies
         if: steps.cache.outputs.cache-hit != 'true'
         run: |
-          pip install -r requirements/required.txt -r requirements/datasets.txt -r requirements/tests.txt
+          pip install -r requirements/required.txt -r requirements/datasets.txt -r requirements/models.txt -r requirements/tests.txt
           pip cache purge
       - name: List pip dependencies
         run: pip list
```

Comment:
Can we change the `datasets` test name to be `optional` or something like that? We are trying to ensure that all tests pass even if optional deps are missing. The name `datasets` was never very clear about this. Maybe `optional` isn't clear either, it makes it sound like it isn't required to pass.

Reply:
I think optional makes the most sense just because that's how it's always defined in pyproject.toml files